public/ics/calendar.php

```diff
@@ -268,6 +268,46 @@
     }
 }
 
+if(isset($absences)){
```

Comment:
Ajouter la condition `and $get_absences`
et en ligne 60 : `$get_absences = filter_input(INPUT_GET, 'absences', FILTER_SANITIZE_NUMBER_INT);

De cette façon, nous ne changerons pas le comportement actuel pour ceux qui ne souhaitent pas intégrer les absences dans les exports ICS.
Pour ceux qui le souhaitent, il suffira s'ajouter &absences=1 dans l'URL